### DISCUSSION TITLE
Add multiple icons and marker icons inside activity

### DISCUSSION THREAD

**Discussion by shailendra-kbtechnology** [clientIO/joint#2657] [Q&A]
### Introduction

Add multiple icons and marker icons inside activity

### Steps to reproduce

Hello Jointjs,

We are implementing multiple icons inside the activity 
Like this:
![Screenshot from 2024-05-10 17-14-55](https://github.com/clientIO/joint/assets/156991175/0848c735-4c59-43b6-8e39-9416e4376499)

My Create activity code 

```ts
    //Add DataBase icon
    shapes.bpmn2.Activity.ACTIVITY_MARKER_ICONS["databaseIcon"] =
      '/databaseicon.svg';
    const activity = new shapes.bpmn2.Activity({
      position: { x, y },
      size: {
        height: WIZARDACTIVITYDEFAULT.HEIGHT,
        width: WIZARDACTIVITYDEFAULT.WIDTH,
      },
      attrs: {
        label: {
          text: labelText,
          fill: WIZARDACTIVITYDEFAULT.FONT,
          fontFamily: WIZARDACTIVITYDEFAULT.FONTNAME,
          fontSize: WIZARDACTIVITYDEFAULT.FONTSIZE,
          textWrap: {
            width: -WIZARDACTIVITYDEFAULT.TEXTWRAPWIDTH,
            height: -WIZARDACTIVITYDEFAULT.TEXTWRAPHEIGHT,
          },
          fontWeight: "normal",
          cursor: !showZoom ? "pointer" : "all-scroll",
          lineHeight: 16,
        },
        background: {
          fill: WIZARDACTIVITYDEFAULT.BACKGROUND,
          cursor: !showZoom ? "pointer" : "all-scroll",
        },
      },
    });
    activity.attr("icon/iconType", ["databaseIcon"]);
    activity.addTo(graph.current!);
```

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
`iconType` does not accept an array of icons.

You can add multiple markers with [`markers/iconTypes`](https://resources.jointjs.com/docs/rappid/v4.0/shapes.bpmn2.html#Activity.attributes.iconTypes) and only a single icon with [`icon/iconType`](https://resources.jointjs.com/docs/rappid/v4.0/shapes.bpmn2.html#Activity.attributes.iconType).

If you want to add a list of icons to an element, check out [List](https://resources.jointjs.com/docs/jointjs/v4.0/joint.html#highlighters.list) highlighter.
